torch/csrc/jit/passes/python_print.cpp

```diff
@@ -242,7 +242,6 @@ struct PythonPrintPass {
   bool isConstantLike(Node* n) {
```

Comment:
We can probably remove this function now since it just checks n->kind() == prim::Constant